### DISCUSSION TITLE
Displaying linker charts in Databricks

### DISCUSSION THREAD

**Discussion by mmagoffin-sd** [moj-analytical-services/splink#1951] [Q&A] [answered]
Hi all,

I'm trying to use the linker.match_weights_chart() method in a Databricks notebook, but when I try to run it I get no chart and no error to point to why the chart isn't displaying. Does anyone know anything about displaying splink charts in Databricks and whether there are some extra settings I could use to be able have the chart functions as expected?

Thanks!

**Comment by mmagoffin-sd** [comment]
*context: I'm using splink==3.8.1 and a cluster with Databricks runtime 12.2 LTS, am able to run record linkage but not see charts!

**Reply by RobinL** [reply, marked as answer]
The charts are Altair charts, so if they're not rendering in databricks it's an Altair issue rather than a Splink issue.

If you assign the chart to a variable like:
chart = linker.match_weights_chart()

Then you can do 
chart.save("myfile.html")

And if you open the file you can see the chart.  

You do need an internet connection to see the chart. If your environment doesn't have one, you can use https://github.com/moj-analytical-services/splink/blob/73f13383c4ec272e719d0d219ca26d56e58a7191/splink/charts.py#L53

**Reply by mmagoffin-sd** [reply]
That worked! Specifically in my case I need to use the save_offline_chart func
